sql-prepared-plan-cache.md

```diff
@@ -169,6 +171,25 @@ mysql> show warnings;
 1 row in set (0.00 sec)
 ```
 
+### Use `Statements Summary` to diagnose
+
+The `Statements Summary` table contains two fields, `plan_cache_unqualified` and `last_plan_cache_unqualified_reason`, which indicate the number of times and the reason why the corresponding query is unable to use the plan cache. You can use these two fields for diagnostic purposes:
```

Comment:
If we can change the `last_plan_cache_unqualified_reason` to `plan_cache_unqualified_last_reason`, it makes more sense to me ;). @qw4990

Reply:
updated